tests/acceptance/features/bootstrap/OCSContext.php

```diff
@@ -310,17 +255,18 @@ public function administratorSendsToOcsApiEndpointWithHeaders(
 		string $url,
 		TableNode $headersTable
 	):void {
-		$this->userSendsToOcsApiEndpointWithHeaders(
-			$this->featureContext->getAdminUsername(),
+		$user = $this->featureContext->getAdminUsername();
+		$password = $this->featureContext->getPasswordForUser($user);
+		$this->userSendsToOcsApiEndpointWithHeadersAndPassword(
```

Comment:
can this method be utilize here?
https://github.com/owncloud/ocis/blob/5c7c363e2bb7f550760455cb93b0a24202cc2970/tests/acceptance/features/bootstrap/OCSContext.php#L119
```suggestion
		$this->sendRequestToOcsEndpoint(
```